shell.nix

```diff
@@ -40,7 +40,10 @@ in haskell.packages.shellFor {
     dev.scripts.fixStylishHaskell
     dev.scripts.fixPurty
     dev.scripts.updateClientDeps
-    pkgs.rPackages.plotly # for generating R plots locally
     pkgs.python3 # for @reactormonk who's running a python web server
-  ];
+  ] ++ (with pkgs; stdenv.lib.optionals (!stdenv.isDarwin) [
```

Comment:
Nit, but we generally avoid `with` these days (hard to see where names come from). And you only use it twice.